src/main/java/Todo.java

```diff
@@ -0,0 +1,12 @@
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public char getTypeIcon() {
+        return 'T';
+    }
+
```

Comment:
Again, consistency - usually your last closing brace does not have an extra empty line above it